### DISCUSSION TITLE
Lessons could use improved titles

### DISCUSSION THREAD

**Discussion by thatblindgeye** [TheOdinProject/curriculum#25446] [Ideas]
This mainly refers to the JavaScript lessons in Foundations, though could be a curriculum-wide thing to keep in mind.

Instead of having several lessons that are "X Part 1", "X Part 2", etc., I believe these lessons should have more descriptive naming. This would actually tell users immediately what the lesson is about when starting them, but also make it a lot easier to navigate back to a previous lesson at a glance should they need to. A rough example of the kind of result I'm envisioning (_old lesson title_ => _new lesson title_):

* Fundamentals Part 1 => Variables and Operators
* Fundamentals Part 2 => Data Types and Conditionals

I will say that using "Part 1", "Part 2", etc. modifiers to lessons I think does have its place, but I think the JS Fundamental lessons go too far with them.

It may not be as simple as renaming the lessons as they *might* need a little restructuring to avoid too much going into the titles themselves and the lesson content (Fundamentals 4, for example, might end up being titled "Arrays and TDD Basics" or something). For that reason I'm assigning the "blocked" label **to prevent anyone from asking to be assigned or working on this for now**, as this may need a discussion to be had with other maintainers before implementing any sort of change.

**Comment by KevinMulhern** [comment]
Moving this to a discussion because it isn't actionable (yet)

**Comment by impronen** [comment]
I like this idea.

Recently I've few times had to scroll through the fundamentals lessons to see what is covered in which, to be able to point someone to the point where this or that is covered. 

If the names would be descriptive, finding the correct spot would be easier.

**Comment by Justin-Bento** [comment]
I was going to write a post about this as well but thankfully I noticed a discussion. 

While going through The Odin Project and making notes for myself, I noticed something strange about Part 2 of the Fundamentals section. There's no headline for Data Types, and this caught my attention. In the lesson overview, the first lesson titled "Name the eight data types in JavaScript" is missing from the curriculum. This omission can confuse readers who prefer to skim through content first before delving into articles.

Instead of a full lesson, the headline could be something like: "Quick Overview of Common Data Types." The headline will include a description what are data types, then include the common data types and a link to the article. This would provide a clearer structure for readers and allow users to choose to search into more depth if they desire.
